src/changelog-updater.ts

```diff
@@ -71,7 +71,7 @@ function buildEntryLine(entry: DependabotEntry): string {
 }
 
 function buildEntryLineStart(entry: DependabotEntry): string {
-  return `- Bumps \`${entry.package}\` from`
+  return `- Bump \`${entry.package}\` from`
```

Comment:
I think this makes a lot of sense.

I wonder if this would be worthing making a configuration option for other users?